<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TestPlot</class>
 <widget class="QWidget" name="TestPlot">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>547</width>
    <height>510</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="TaurusPlot" name="taurusPlot">
     <property name="allowZoomers">
      <bool>false</bool>
     </property>
     <property name="enableMagnifier" stdset="0">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>100</height>
      </size>
     </property>
     <property name="title">
      <string>Cursors</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="1" column="1">
       <widget class="TaurusValueLineEdit" name="taurusValueLineEditVert"/>
      </item>
      <item row="1" column="2">
       <widget class="TaurusValueLineEdit" name="taurusValueLineEditTimeMin"/>
      </item>
      <item row="1" column="3">
       <widget class="TaurusValueLineEdit" name="taurusValueLineEditTimeMax"/>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Vertical</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Time min</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Time max</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TaurusValueLineEdit</class>
   <extends>QLineEdit</extends>
   <header>taurus.qt.qtgui.input</header>
  </customwidget>
  <customwidget>
   <class>TaurusPlot</class>
   <extends>QwtPlot</extends>
   <header>taurus.qt.qtgui.plot</header>
  </customwidget>
  <customwidget>
   <class>QwtPlot</class>
   <extends>QFrame</extends>
   <header>qwt_plot.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
